sdks/python/apache_beam/typehints/opcodes.py

```diff
@@ -35,7 +35,7 @@
 import types
 from functools import reduce
 
-from apache_beam.typehints import row_type
+from apache_beam.typehints import row_type, trivial_inference
```

Comment:
Beam style is one import per line
```suggestion
from apache_beam.typehints import row_type
from apache_beam.typehints import trivial_inference
```